betterproto/tests/util.py

```diff
@@ -60,3 +63,25 @@ def get_test_case_json_data(test_case_name, json_file_name=None):
 
     with open(test_data_file_path) as fh:
         return fh.read()
+
+
+def find_module(
+    module: ModuleType, predicate: Callable[[ModuleType], bool]
+) -> Optional[ModuleType]:
+    if predicate(module):
+        return module
+
+    module_path = pathlib.Path(*module.__path__)
+
+    for sub in list(module_path.glob("**/")):
```

Comment:
does the trailing slash mean something?

Reply:
err nope. it was an attempt to match only dirs, but that didn't work. thanks!
removed~ ✔️